### DISCUSSION TITLE
Suggestions for buttons.

### DISCUSSION THREAD

**Discussion by Kowlin** [Cog-Creators/Red-DiscordBot#5088] [Ideas]
With the recent release of buttons we're looking for interesting ways to implement these across Red. If you have any ideas do let us know by shooting an reply to this discussion.

**Comment by Jacob-1999** [comment]
Can make some repetitive/tedious setup commands interactive. ModLog, Mutes, Roles, Admin/Mod Roles, etc.

**Reply by lincolnthedev** [reply]
+1

**Comment by Danstr5544** [comment]
I think it could have some value in improving the usage of `[p]warnings` by changing it to be a more detailed page for each warning, with buttons to, for instance, remove a warning without need of another command.
